docs/getting-started.md

```diff
@@ -0,0 +1,140 @@
+﻿
+# Kubevirtci
+
+Kubevirtci rule is to create a pre provisioned clusters, publish them as a container image,\
+and allow people to run a containerized version of a k8s / open-shift cluster.
+
+Target audience of this project is Developers of kubevirt.
+
+## Getting started
+Download this repo
+```
+git clone https://github.com/kubevirt/kubevirtci.git
+cd kubevirtci
+```
+
+Make sure you have jq installed (can be installed by yum/dnf with the basic repositories).
+
+#### gocli developer guide:
+gocli is a binary that assist in provisioning and spinning (cluster-up) a cluster.
+Uses bazel as build tool.
```

Comment:
I don't think bazel is that important here. Maybe we don't need to mention it at all. Definitely not as one of the first parts about kubevirtci. All I need to use are `make` targets anyway, no?